contracts/PangolinExchange.sol

```diff
@@ -89,6 +90,50 @@ contract PangolinExchange is Ownable, IAssetsExchange, ReentrancyGuardUpgradeabl
     return true;
   }
 
+
+  /**
+   * Adds supported asset
+   * @dev _asset asset to be added
+  **/
+  function setAsset(bytes32 _asset, address _address) external override onlyOwner {
+    require(_asset != "", "Cannot set an empty string asset.");
```

Comment:
How about duplicate asset name, will it be updated or throw an error?

Reply:
it will be updated

Reply:
i will make it more clear in comment